### PR TITLE
[pcre/fr] PCRE French translation

### PR DESCRIPTION
**Pull request by Th3G33k**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [ ] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [ ] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [ ] Yes, I have double-checked quotes and field names!

Added French translation for PCRE

### CODE REVIEW

File: fr-fr/pcre-fr.html.markdown [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+filename: pcre-fr.txt
+contributors:
+    - ["Sachin Divekar", "http://github.com/ssd532"]
+translators:
+    - ["Th3G33k", "https://github.com/Th3G33k"]
+lang: fr-fr
+
+---
+
+Une expression régulière (regex ou regexp en abrégé) est une chaîne de texte spéciale pour décrire un masque de recherche. Par exemple, pour extraire le nom de domaine d'une chaîne, nous pouvons dire `/^[a-z]+:/` et cela correspondra à `http:` à partir de `http://github.com/`.
```
Comment (vendethiel) [inline fr-fr/pcre-fr.html.markdown:12]: c'est le protocole, pas le nom de domaine

File: fr-fr/pcre-fr.html.markdown [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+  -      indique une plage de caractères
+  [      classe de caractères POSIX (uniquement si elle est suivie d'une syntaxe POSIX)
+  ]      termine la classe de caractères
+```
+
+PCRE fournit des types de caractères génériques, également appelés classes de caractères.
+
+```
+  \d     tout chiffre décimal
+  \D     tout chiffre non décimal
```
Comment (vendethiel) [inline fr-fr/pcre-fr.html.markdown:52]: tout sauf un chiffre décimal*

File: fr-fr/pcre-fr.html.markdown [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+  \h     tout caractère d'espace horizontal blanc
+  \H     tout caractère, autre que l'espace horizontal blanc
+  \s     tout caractère d'espace blanc
+  \S     tout caractère, autre que l'espace blanc
+  \v     tout caractère d'espace vertical blanc
+  \V     tout caractère, autre que l'espace vertical blanc
+  \w     tout caractère pouvant former un mot (lettre, chiffre, underscore)
+  \W     tout caractère, ne pouvant pas former un mot
```
Comment (vendethiel) [inline fr-fr/pcre-fr.html.markdown:60]: pas de `,` sur les exclusions

File: fr-fr/pcre-fr.html.markdown [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+## Exemples
+
+Nous allons tester nos exemples sur la chaîne sujet suivante :
+
+```
+66.249.64.13 - - [18/Sep/2004:11:07:48 +1000] "GET /robots.txt HTTP/1.0" 200 468 "-" "Googlebot/2.1"
+```
+
+ It is a standard Apache access log.
```
Comment (vendethiel) [inline fr-fr/pcre-fr.html.markdown:71]: Non traduit

**Review by vendethiel — CHANGES_REQUESTED** [review]

**Comment by Th3G33k** [comment]
I applied the suggestions, thank you @vendethiel

**Review by vendethiel — APPROVED** [review]

**Comment by vendethiel** [comment]
Merci !

**Pull request merged**